command/v7/continue_deployment_command.go

```diff
@@ -0,0 +1,79 @@
+package v7
+
+import (
+	"code.cloudfoundry.org/cli/command/flag"
+	"code.cloudfoundry.org/cli/command/v7/shared"
+)
+
+type ContinueDeploymentCommand struct {
+	BaseCommand
+
+	RequiredArgs    flag.AppName `positional-args:"yes"`
```

Comment:
**question/non-blocking** Do we have/want a `--no-wait` flag here?

Reply:
No, it wasn't mentioned in the story. I can add it, if needed.

Reply:
Doesn't need to block this PR. So maybe a follow up discussion + issue?

Reply:
Created internal issue for this.